src/styles/theme.js

```diff
@@ -7,7 +7,8 @@ const palette = {
   mauve: '#a05258',
   white: '#ffffff',
   grey: '#aaa',
-  dark_gray: '#333333',
+  light_grey: '#ccc',
```

Comment:
Feels weird to see snake case, rather than kebab (like our CSS) or camel case (like our JS).

Reply:
I agree... I'll change that too. Good catch.